src/library_syscall.js

```diff
@@ -222,6 +223,16 @@ var SyscallsLibrary = {
       else assert(high === -1);
 #if SYSCALL_DEBUG
       err('    (i64: "' + low + '")');
+#endif
+      return low;
+    },
+    // Like get64, but read high first and low second.
+    get64hilo: function() {
```

Comment:
how about `reversed` or `highLow` instead of `hilo`?

Reply:
Fixed; changed to `reversed`.